core/scss/utilities/mixins/animation/index.scss

```diff
@@ -0,0 +1,11 @@
+@charset "UTF-8";
+
+//
+// Animation
+//
+// Mixins for adding animations
+//
+// Style guide: Mixins.Animation
+//
+
+@import 'fancy-hover';
```

Comment:
I like this new directory. Makes sense to have a collection of these similar things.